subprojects/packagefiles/opencl-headers/meson.build

```diff
@@ -0,0 +1,16 @@
+project(
+  'OpenCLHeaders',
+  'c',
+  license : 'Apache-2.0',
+  version : '2021.06.30',
+)
+
+pkgc = import('pkgconfig')
+pkgc.generate(name: 'OpenCLHeaders',
+  version: '2.2',
+  description: 'C language headers for the OpenCL API'
+)
+
+opencl_headers_dep = declare_dependency(
+  include_directories : include_directories('CL'),
```

Comment:
This should declare the version as well, since it defaults to the project version which is not what we want to use here.

That way, `dependency('OpenCLHeaders', version: '==2.2')` will work correctly.

Reply:
Done.